packages/css-engine/src/core/to-value.ts

```diff
@@ -91,12 +91,21 @@ export const toValue = (
   }
 
   if (value.type === "layers") {
-    return value.value
-      .map((value) => toValue(value, transformValue))
+    const valueString = value.value
+      .map((layer) => toValue(layer, transformValue))
+      .filter((value) => value !== "none")
       .join(", ");
+    return valueString === "" ? "none" : valueString;
   }
 
   if (value.type === "tuple") {
+    if (value.hidden) {
```

Comment:
If none as you wrote above has no sense, do not return it.

Reply:
I will take a look, the same one is being used for gradients too. So, will refactor without effecting gradients 👍

Reply:
not sure, why you saying none doesn't make sense? its the same comment as you have made above for background-image

Reply:
yup. but changing the filtering to be done using `hidden` field now. Should be more reliable